it/mail/chrome/messenger-smime/msgSecurityInfo.properties

```diff
@@ -27,43 +27,43 @@ SIHeaderMismatch=L'indirizzo e-mail del firmatario indicato nel certificato è d
 SICertWithoutAddress=Il certificato utilizzato per la firma del messaggio non contiene un indirizzo di posta elettronica. Controllare i dettagli del certificato di firma per capire chi ha firmato il messaggio.
 
 ## Encryption Information strings
-EINoneLabel=Messaggio non cifrato
-EINone=Questo messaggio non è stato cifrato prima di essere inviato. Le informazioni inviate tramite Internet senza cifratura possono essere lette da altre persone durante il transito.
+EINoneLabel=Messaggio non crittografato
+EINone=Questo messaggio non è stato crittografato prima di essere inviato. Le informazioni inviate tramite Internet senza crittografia possono essere lette da altre persone durante il transito.
 
-EIValidLabel=Messaggio cifrato
-EIValid=Questo messaggio è stato cifrato prima di essere inviato. La cifratura rende molto difficile per altre persone leggere le informazioni in transito attraverso la rete.
+EIValidLabel=Messaggio crittografato
+EIValid=Questo messaggio è stato crittografato prima di essere inviato. La crittografia rende molto difficile per altre persone leggere le informazioni in transito attraverso la rete.
 
 EIInvalidLabel=Il messaggio non può essere decifrato.
-EIInvalidHeader=Questo messaggio è stato cifrato prima di essere inviato, ma non può essere decifrato.
+EIInvalidHeader=Questo messaggio è stato crittografato prima di essere inviato, ma non può essere decifrato.
 
 EIContentAltered=Il contenuto del messaggio sembra essere stato alterato durante la trasmissione.
-EINoDecryptCert=The certificate used to encrypt the message cannot be found.
-EIPasswordError=You did not enter your Master Password correctly.
-EIInvalidCipher=The message was encrypted using an encryption strength that this version of your software does not support.
-EIClueless=Si sono verificati problemi sconosciuti con questo messaggio cifrato.
+EINoDecryptCert=Il certificato utilizzato per crittografare il messaggio non può essere trovato.
+EIPasswordError=Non hai inserito correttamente la tua Password principale.
+EIInvalidCipher=Il messaggio è stato crittografato utilizzando una crittografia la cui forza non è supportata da questa versione del tuo software.
+EIClueless=Si sono verificati problemi sconosciuti con questo messaggio crittografato.
 
 ## Signing Power Information string
-SPCanLabel=Message Can Be Signed
-SPCanHeader=You have chosen to digitally sign this message before sending it.
-SPCan=When other people receive your signed message, they can verify that the message comes from you and that it has not been altered since you signed it.
+SPCanLabel=Il messaggio può essere firmato
+SPCanHeader=Hai scelto di firmare digitalmente questo messaggio prima d'inviarlo.
```

Comment:
CORRECTION NEEDED.

Please replace "d'" with "di " (a blank space is required between "di" and "inviarlo").